style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", serif;
+}
+
+.container {
+  height: 718px;
+  width: 375px;
+  left: 0;
+  top: 0;
+  border-bottom-left-radius: 120px;
+  background-color: white;
+  border: 5px solid white;
+  background-image: url(./images/header-shapes\ mobile.png);
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  border-bottom-left-radius: 50%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo {
+  padding-top: 20px;
+  padding-left: 30px;
+  color: #6070ff;
+  font-weight: bold;
+}
+
+.union {
+  padding-top: 25px;
+  padding-right: 30px;
+}
+
+.content {
+  padding-left: 20px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
- [ ] I see that you use `font-family` multiple times across your CSS stylesheet. It would be a good idea to find a way to avoid this code repetition to keep your code clean and readable at all times. 🤓 

For example, you could create a class `font-poppins` with this style:
```CSS
.font-poppins {
  font-family: 'Poppins', serif;
}
```

and then, use it in your HTML elements that use it like this:

```HTML
<div class="font-poppins">Hello world</div>
```